hw/top_earlgrey/data/ip/chip_rom_ctrl_testplan.hjson

```diff
@@ -21,8 +23,11 @@
               the ROM. Verify that we can successfully power up the chip in this case.
             - In PROD LC state, verify that the pwrmgr does not fully power up if the computed
               digest does not match the top 8 words of the ROM.
+
```

Comment:
Maybe add some notes for silicon targets that the prod modes can be tested in non-secure environments and non-prod stages will need to be done in a secure site.

Reply:
Hmm, it's worse than that: something has gone horribly wrong if the expected and measured digests don't match (and we can probably give up on bring-up at that point!)

I guess it's still worth checking that the digests are sensibly readable by software. I'll add some notes about that and re-push. Thanks!